decidim-forms/app/packs/entrypoints/decidim_questionnaire_answers_pdf.scss

```diff
@@ -1 +0,0 @@
-@import "stylesheets/decidim/forms/questionnaire-answers-pdf.scss";
```

Comment:
I see that this file is used in `./decidim-forms/app/packs/entrypoints/decidim_forms.js` but at least by grepping by the classes names (like `.questionnaire-answers`) it seems like it is no being used anymore. Can you delete it (and remove it from that file too) 🙏🏽 ?

Reply:
Fixed in [1b8a59c](https://github.com/decidim/decidim/pull/13617/commits/1b8a59c8c929ce9756c3e9906e53a9ef54132e10)